### PR TITLE
[DOMA-4250] 🩼 Global apps features🩼 

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

File: apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsContainer.tsx [hunk 1]
```diff
@@ -66,6 +74,20 @@ export const GlobalAppsContainer: React.FC = () => {
         iframeRefs.current = iframeRefs.current.slice(0, appUrls.length)
     }, [appUrls])
 
+    useEffect(() => {
+        const globalFeatures = objs.reduce((acc, app) => {
```
Comment (sitozzz) [inline apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsContainer.tsx:78]: What about this?
```suggestion
        const globalFeatures = objs.reduce((featureNames, app) => {
```

Comment (SavelevMatthew) [reply apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsContainer.tsx]: added more clear name 👍

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsFeaturesContext.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    features: IFeaturesType
+    registerFeatures: IRegisterFeaturesType
+    requestFeature: IRequestFeatureAction
+    addFeatureHandler: (handler: IRequestFeatureHandler) => void,
+    removeFeatureHandler: (handler: IRequestFeatureHandler) => void,
+}
+
+const GlobalAppsFeaturesContext = createContext<IGlobalAppsFeaturesContext>({
+    features: {},
+    registerFeatures: () => ({}),
```
Comment (sitozzz) [inline apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsFeaturesContext.tsx:34]: May this types been specified to correspond with `FeaturesEmitter` object declared bellow?

File: apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsContainer.tsx [hunk 3]
```diff
@@ -35,6 +39,7 @@ type ModalInfo = {
 const MUTATION_RESULT_MESSAGE_NAME = 'CondoWebUserEventResult'
 const MODAL_OPEN_RESULT_MESSAGE_NAME = 'CondoWebOpenModalResult'
 const MODAL_CLOSE_RESULT_MESSAGE_NAME = 'CondoWebCloseModalResult'
+const REQUEST_FEATURE_MESSAGE_NAME = 'CondoWebRequestFeatureRequest'
```
Comment (sitozzz) [inline apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsContainer.tsx:42]: Strange constant body)

File: apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsFeaturesContext.tsx (hunk 2 above)
Comment (SavelevMatthew) [reply apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsFeaturesContext.tsx:34]: That's not type, it's default implementation for case, when there's no Context provider. (For example in CC)

File: apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsContainer.tsx (hunk 3 above)
Comment (SavelevMatthew) [reply apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsContainer.tsx]: Yep, it was a typo

**Pull request merged**